components/browser/search/src/main/java/mozilla/components/browser/search/SearchEngineManager.kt

```diff
@@ -28,6 +28,7 @@ class SearchEngineManager(
 ) {
     private var deferredSearchEngines: Deferred<List<SearchEngine>>? = null
     private val scope = CoroutineScope(Dispatchers.Default)
+    var defaultSearchEngine: SearchEngine? = null
```

Comment:
I think in `SearchEngineManager` it may be good to still use the name and only have a setter for it (e.g. `setDefault(name: String)`) Otherwise it may be confusing if you look at it from the outside and you have two way to get to a `SearchEngine` - calling `getDefaultSearchEngine` but also `defaultSearchEngine`. But the second one only returns something you have set on the manager manually and not really the default in all cases. :)